core/ui-session/src/session.ts

```diff
@@ -0,0 +1,386 @@
+/*
+  Copyright (C) 2018-present evan GmbH.
+
+  This program is free software: you can redistribute it and/or modify it
+  under the terms of the GNU Affero General Public License, version 3,
+  as published by the Free Software Foundation.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see http://www.gnu.org/licenses/ or
+  write to the Free Software Foundation, Inc., 51 Franklin Street,
+  Fifth Floor, Boston, MA, 02110-1301 USA, or download the license from
+  the following URL: https://evan.network/license/
+*/
+
+import { config, routing } from '@evan.network/ui-dapp-browser';
+import {
+  ExecutorAgent,
+  logLog,
+  logLogLevel,
+  Runtime,
+} from '@evan.network/api-blockchain-core';
+
+import lightwallet from './lightwallet';
+import bccHelper from './bccHelper';
+
+// external executor variables
+let agentExecutor;
+// save the current account for later usage
+let lastAccount = '';
+// save the current account for later usage
+let lastIdentity = '';
+// used to check for url overwritten account / identity parameters
+const queryParams = routing.getQueryParameters();
```

Comment:
we should really decouple this from external stuff like routing instances. 
Better pass 'queryParams' within the constructor or use method like 'setQueryParams'